dive_sailthru_client/tests/test_integration.py

```diff
@@ -61,6 +66,9 @@ def test_update_job_with_filename(self):
             os.unlink(f.name)
         # now check if it really updated
         test_updated_var = self._get_user_var(self.test_email, self.test_var_key)
+        test_updated_var = unicodedata.normalize('NFKD', test_updated_var).encode('ascii', 'ignore')
+        test_updated_var = datetime.datetime.strptime(test_updated_var[12:], '%Y-%m-%d %H:%M:%S.%f').strftime('%Y-%m-%d %H:%M:%S')
+        test_updated_var = 'start value %s' % test_updated_var
```

Comment:
This is like the inverse of the above. Not sure why we're converting a string to a datetime then back to a string.

Reply:
all the same stuff; ignore please.  reverted.